### DISCUSSION TITLE
Add .dist().plot() method to pymc distributions

### DISCUSSION THREAD

**Discussion by ezDataCode** [pymc-devs/pymc3#7306] [Ideas]
It would be nice to quickly visualize a pymc distribution. [This video ](https://www.youtube.com/watch?v=tKJDIN4Y7LE) and [this notebook](https://github.com/pymc-labs/research/blob/main/youtube-videos/distribution_plotting.ipynb) describes how it can be done. Since this is a common situation with a known solution, I think this feature should be added as a built-in .dist().plot() method of all pymc distributions. Any reason not to add this as a standard feature?

**Comment by ricardoV94** [comment]
The API since PyMC3 (which that example uses). Something like `pm.plot_dist(rv)` would make more sense. Replace `self.draw` (which no longer exists) in that example by `pm.draw(rv, draws=1000)`

**Comment by aloctavodia** [comment]
No strong opinion about this just wanted to mention two points.

This idea was already proposed and rejected, mainly argument was something along the lines of not (re)adding plotting code to pymc. Of course this can be revisited.

In the meantime, or as alternative you can use preliz to visualize distributions. It has methods for pdf, cdf and ppf, and also an interactive method to change parameters with sliders. https://preliz.readthedocs.io/en/latest/examples/param_space_1d_examples.html

**Reply by ricardoV94** [reply]
Does preliz give a mapping between its parametrizations and the ones PyMC uses?

**Reply by aloctavodia** [reply]
PreliZ uses the same parametrization used by PyMC, including alternative parametrization.
